client/homebrew/editor/editor.jsx

```diff
@@ -437,7 +447,7 @@ const Editor = createClass({
 					onChange={this.props.onBrewChange('text')}
 					editorTheme={this.state.editorTheme}
 					rerenderParent={this.rerenderParent}
-					style={{  height: `calc(100% - ${this.state.snippetbarHeight}px)` }} />
+					style={{  height: `calc(100% - 26px)` }} />
```

Comment:
There is some conflict going on here where we have two different sources trying to edit the height of the .codeMirror component. One is the resizeObserver, and then again here in renderEditor we immediately overwrite that value.